autrainer/datasets/utils/dataset_wrapper.py

```diff
@@ -17,9 +17,9 @@ def __init__(
         features_subdir: str,
         index_column: str,
         target_column: Union[str, List[str]],
-        file_type: str,
         file_handler: AbstractFileHandler,
         df: pd.DataFrame,
+        file_type: str = None,
```

Comment:
It makes sense to not overlay the `file_type` if it should be the default one in the dataframe anyways. But for this to have the necessary effect we need in the preprocessing script (e.g., drop it from the dataset configuration to suppress overlying the type), we should make this explicitly nullable (`Optional[str]`).
This would mean that the `file_type` will no longer be required in dataset configurations without preprocessing.
Right now its kind of a hidden control flow that its nullable which could lead to interesting masked errors when relying on the file type when implementing custom datasets.

Reply:
Fixed